client/scss/components/forms/_field-comment-control.scss

```diff
@@ -40,22 +37,45 @@ $button-size: theme('spacing.5');
     .tab-content--comments-enabled .w-field__input:focus-within > &,
     &:hover,
     &:focus,
-    &.w-field__comment-button--focused {
+    &.w-field__comment-button--focused,
+    .w-field--admin_model_chooser.w-field--commentable:hover & {
       visibility: visible;
       opacity: 1;
     }
   }
+
+  // Position comment icons in choosers beside the other actions
+  .tab-content--comments-enabled .chooser & {
+    position: unset;
+    inset-inline-end: 0;
+    margin-inline-start: 0;
+    transform: translateY(3px);
+  }
+
+  .tab-content--comments-enabled .unchosen:hover &,
+  .tab-content--comments-enabled .unchosen:focus-within &,
+  .tab-content--comments-enabled .chooser .chosen & {
+    opacity: 1;
+    visibility: visible;
+  }
 }
 
-// Add extra space to the right of commentable fields for the buttons.
 .w-field--commentable {
-  .tab-content--comments-enabled & .w-field__input {
-    padding-inline-end: $button-size;
+  .w-field__input {
+    padding-inline-end: calc($button-size + $button-space);
```

Comment:
Could we refactor this to always be the case, even for fields that aren’t commentable? It feels odd to me the two scenarios have different right padding.

Reply:
Done! I've added this to field.scss so it can be more apparent there that this padding is applied to all